spec/factories/entities.rb

```diff
@@ -36,6 +36,16 @@
     name 'Human Being'
     primary_ext 'Person'
   end
+
+  factory :merge_source_person, parent: :entity_person do
```

Comment:
i didn't know you could inherit from other factories using parent...cool

Reply:
yeah! i had fun with factories on this card. :D